app/test/marionette/home_test.js

```diff
@@ -0,0 +1,27 @@
+var BuddyUp = require('./lib/buddyup'),
+  assert = require('chai').assert;
+
+marionette('Home view', function() {
+
+  var app;
+  var homeView;
+  var client = marionette.client();
+
+  setup(function(done) {
+    app = new BuddyUp(client);
+    app.launch()
+      .then(function(home) {
+        homeView = home;
+        done()
+      });
+  });
+
+  test('should open the "ask a question" panel', function(done) {
+    homeView.openAskQuestion()
+      .then(function(questionView) {
```

Comment:
Nit: I prefer to not over-indent promises because it feels like something is missing when we close them. Preferred style:
```js
foo().then(function() {
});
```
or
```js
foo()
.then(function() {
});
```